kano_video/logic/youtube.py

```diff
@@ -5,16 +5,28 @@
 # Copyright (C) 2014-2015 Kano Computing Ltd.
 # License: http://www.gnu.org/licenses/gpl-2.0.txt GNU General Public License v2
 #
-# Interracts with the YouTube API to retreive video links
+# Interracts with the YouTube API v3.0 to retreive video links
 #
 
 import os
 from shutil import rmtree
 from kano.utils import requests_get_json, run_cmd
 from kano.logging import logger
+from apiclient.discovery import build
+from apiclient.errors import HttpError
+from oauth2client.tools import argparser
+from isodate import parse_duration # Used to process the ISO 8601 duation format used for the video length
+
+DEVELOPER_KEY = "" # Insert your public api key here from console.developers.google.com
```

Comment:
Can this be moved into a separate module so that it is easier to provide separately?